rust_dev_preview/cross_service/rest_ses/src/configuration.rs

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Environment and settings loading.
+//!
+//! Loads environment specific details for ARNs, tcp listeners, etc.
+use std::net::{IpAddr, SocketAddr};
+
+use color_eyre::Report;
+use secrecy::Secret;
+use serde::Deserialize;
+
+use crate::client::{Arn, Email};
+
+/// Top level settings, for organization.
+#[derive(Debug, Deserialize)]
+pub struct Settings {
+    pub log_level: String,
+    pub application: ApplicationSettings,
+    pub rds: RdsSettings,
+    pub ses: SesSettings,
+}
+
+impl Settings {
+    pub fn address(&self) -> SocketAddr {
+        SocketAddr::new(self.application.address, self.application.port)
+    }
+}
+
+/// Application specific settings, for the HTTP server.
+#[derive(Debug, Deserialize)]
+pub struct ApplicationSettings {
+    pub name: String,
+    pub address: IpAddr,
+    pub port: u16,
+}
+
+/// Settings for the RDS client, primarily the DB & Cluster to access.
```

Comment:
RDS client, --> Amazon Relational Database Service (Amazon RDS) client,

For "DB & Cluster", does this refer to a "DB instance and cluster"?